package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -156,6 +153,11 @@ class RDKitParser(TopologyReaderBase):
     .. versionadded:: 2.0.0
     .. versionchanged:: 2.1.0
        Added R/S chirality support
+    .. versionchanged:: 2.8.0
+       Removed type guessing (attributes guessing takes place now
+       through universe.guess_TopologyAttrs() API). If atoms types is not
+       provided, the type attribute get the same values as the element attribute.
```

Comment:
```suggestion
       present in the input rdkit molecule as a _TriposAtomType property,
       the type attribute get the same values as the element attribute.
```
I got myself confused a couple of times over this change, more details might be useful.